book/files-modules-and-programs/README.md

```diff
@@ -153,33 +153,19 @@ itself. [OCaml toolchain/dune]{.idx}
 
 ```sh dir=examples/freq-dune
 $ grep -o '[[:alpha:]]*' freq.ml | ./_build/default/freq.bc
```

Comment:
This looks wrong, and is also what the test is complaining about.  I'm not sure how this change got in here, but it looks like it should be reverted.

Reply:
ugh, I have no idea. Let me fix this